wurst/systems/crafting/QuickMakeSpell.wurst

```diff
@@ -265,6 +267,7 @@ function getQuickMakeSpells() returns LinkedList<QuickMakeSpell>
         new QuickMakeSpell("Spirit Ward"               , TRANSMUTE_TOOLTIP+SPIRIT_WARD_RECIPE        , "AbsorbMagic"               , ABILITY_QM_SPIRIT_WARD        , 2, 2, "S", QM_TOOLIP, "manaflareoff"),
         new QuickMakeSpell("Poison"                    , TRANSMUTE_TOOLTIP+POISON_RECIPE             , "HealingSalve"              , ABILITY_QM_POISON             , 0, 2, "Z", QM_TOOLIP, "manaflareon"),
         new QuickMakeSpell("Ultra Poison"              , TRANSMUTE_TOOLTIP+ULTRA_POISON_RECIPE       , "VialFull"                  , ABILITY_QM_ULTRA_POISON       , 1, 2, "X", QM_TOOLIP, "manashieldoff"),
+        new QuickMakeSpell("Plague Bomb"               , TRANSMUTE_TOOLTIP+PLAGUE_BOMB_RECIPE        , "PlagueBomb"                , ABILITY_QM_PLAGUE_BOMB        , 0, 1, "Q", QM_TOOLIP, "manashieldon"),
```

Comment:
Hotkey should have a consistent layout, I took example on [recent RTS](https://giantbomb1.cbsistatic.com/uploads/original/8/82567/1816719-gridmainqjgb.jpg) when I remapped the hotkeys, fireball scroll is already mapped to Q, plague bomb should be D.